atlp-rwanda/e-commerce-lydia-32-bn
# Delete item from collection

### What does this PR do?
This PR implements the functionality to delete a product from the collection in the database.
### Description of Task to be completed?
1. User haver to be logged in and is of type 'seller'
2. The Seller deleting the product have to be the owner of it(ie. product.userId equals to Id of logged-in user)
3. When no product found error 404(NOT FOUND will be thrown)
4. When trying to delete product yet you are not a seller also error will be thrown
### How should this be manually tested?
1. Register as a seller
2. Verify your account
3. Login with valid credentials
4. Try adding product
5. Finally try deleting added product
#### N/A

### Any background context you want to provide?

#### N/A

### Screenshots (if appropriate)

#### N/A

### Questions:

#### N/A

### What are the relevant pivotal tracker stories
https://www.pivotaltracker.com/story/show/187570876
#### N/A
